docs/source/data/subscriptions.mdx

```diff
@@ -0,0 +1,508 @@
+---
+title: Subscriptions
+description: Persistent GraphQL operations
+---
+
+import {ExpansionPanel} from 'gatsby-theme-apollo-docs';
+
+> **Subscriptions are not supported in [Apollo Federation](https://www.apollographql.com/docs/federation/).**
+
+**Subscriptions** are GraphQL operations that maintain an active connection between client and server. This enables your server to _push_ updates to a subscription's result as they occur, instead of requiring your client to _poll_ for updates.
```

Comment:
I don't think maintaining an active connection is what defines subscriptions. What differentiates them from queries is that they are triggered by events, and they generate a new result for each such event. Although delivering results as soon as they are generated works best over a persistent connection, there is nothing stopping you from implementing subscriptions on top of polling.